src/plugins/intel_cpu/tests/functional/custom/single_layer_tests/classes/activation.cpp

```diff
@@ -235,6 +235,7 @@ std::string ActivationLayerCPUTest::getPrimitiveType(const utils::ActivationType
             (activation_type == utils::ActivationTypes::Clamp) ||
             (activation_type == utils::ActivationTypes::Exp) ||
             (activation_type == utils::ActivationTypes::Floor) ||
+            (activation_type == utils::ActivationTypes::Mish) ||
```

Comment:
Could you please add `Mish` to the list of tested operation `activationTypes()` below?